src/index.js

```diff
@@ -32,28 +32,34 @@ class TelldusTDToolPlatform {
     this.log = log
     this.config = config
     this.homebridge = homebridge
+    this.senzors = config["sensors"] || [];
```

Comment:
I'd really like to avoid this, as sensors can be derived from the config object. Let's keep state to a minimum and rely on the config object instead.